_posts/2020-Q1-open-source-report.markdown

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: 2020-q1-open-source-report
+date: 2020-04-09 20:12:12
+categories: [open-source]
+author: bronzdoc
+
+---
+
+# Open Source at OmbuLabs: 2020 Q1 Report
+
+![ombulabs open source report](https://external-preview.redd.it/zbCp6e3VikD9vsKOtTGVKI90UnWIOZN5NMRNSvgvXxE.jpg?width=960&crop=smart&auto=webp&s=9318c01ae704565ad90f4736aec5af6b671d98fa)
+
+# Relevant News
+
+## Skunk
+
+[Skunk](https://github.com/fastruby/skunk) is a RubyCritic extension to calculate the [SkunkScore](https://www.fastruby.io/blog/code-quality/intruducing-skunk-stink-score-calculator.html) for a file or project.
+
+From Jan 01, 2020 through Apr 05, 2020, fastruby/skunk gained 58 new commits, contributed by 5 authors. There were 506 additions and 269 deletions across 26 files.
```

Comment:
@bronzdoc I find this a bit hard to read. Can you make some refactor to it?

Reply:
🤔 This is auto generated by the reporting tool. I would prefer not to change it